src/main/scala/org/camunda/feel/impl/builtin/TemporalBuiltinFunctions.scala

```diff
@@ -24,7 +25,8 @@ class TemporalBuiltinFunctions(clock: FeelEngineClock) {
     "day of year" -> List(dateTimeFunction(getDayOfYear)),
     "day of week" -> List(dateTimeFunction(getDayOfWeek)),
     "month of year" -> List(dateTimeFunction(getMonthOfYear)),
-    "week of year" -> List(dateTimeFunction(getWeekOfYear))
+    "week of year" -> List(dateTimeFunction(getWeekOfYear)),
+    "last day of the month" -> List(dateTimeFunction(getLastDayOfTheMonth))
```

Comment:
:x: We should rename the function to `last day of month` instead of `last day of the month`.

In general, the name would be fine. But it aligns not so well with the existing function names, like `day of week`, `week of year`.